object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -178,8 +188,15 @@ private void checkPseudoAssignToOwning(Node node) {
       } else {
         throw new BugInCF("unexpected AssignmentContext type " + assignmentContext.getClass());
       }
+    } else {
+      // In this case, we are handling method invocation nodes that are not assigned to a local
+      // variable node but the receiver of the method is in the defs
+      LocalVariableNode mustCallChoiceParam = getMustCallChoiceParam(node);
```

Comment:
_This_ is the check that should be moved to `shouldSkipInvokePseudoAssignCheck` (I think)